sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusClientBuilderTest.java

```diff
@@ -266,55 +262,6 @@ public void testConnectionStringWithSas() {
                 .connectionString("Endpoint=sb://sb-name" + TestUtils.getEndpoint() + "/;EntityPath=sb-name"));
     }
 
-    @Test
```

Comment:
These are valid usecases, we shouldn't delete them. IIRC, we were going to leave these as-is until a solution was found?

Reply:
Yes, we moved this SAS/NamedKey tests to [ServiceBusNonFederatedIntegrationTest](https://github.com/Azure/azure-sdk-for-java/blob/0dd3a66efc5a988282d6e183ff0a9e05d94ca85a/sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusNonFederatedIntegrationTest.java)

Reply:
We moved this AzureSasCredential / AzureNamedKeyCredential
(those are not federated) tests to a new test file [ServiceBusNonFederatedIntegrationTest](https://github.com/Azure/azure-sdk-for-java/blob/0dd3a66efc5a988282d6e183ff0a9e05d94ca85a/sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusNonFederatedIntegrationTest.java)